### DISCUSSION TITLE
mirai-console有计划任务功能的支持吗？

### DISCUSSION THREAD

**Discussion by AdorableParker** [mamoe/mirai-console#266] [问答]
类似于nonebot的计划任务功能，用于定时提醒、动态推送之类的功能上
如果没有原生支持，自行实现这个功能，若使用 `whileSelectMessages`来轮询时间来实现计划任务，是否会有问题？

**Comment by Him188** [comment]
其实可以简单 `launch` 一个协程然后 `delay`
